load_config.py

```diff
@@ -1,6 +1,57 @@
+import sys
+
 import yaml
+import os.path
+import shutil
+
+
+def load_config(file_name):
+    try:
+        with open(file_name) as file:
+            #grab config file
+            config = yaml.load(file, Loader=yaml.FullLoader)
+
+            #since we're here, notify user of testing status so they know if they're using money or not
+            if config['TRADE_OPTIONS']['TEST']:
```

Comment:
This does not belong here and should be moved outside, otherwise you will print the message every time someone loads the config